api/dns/domain_api.go

```diff
@@ -29,7 +29,7 @@ func NewDomainService(concertoService utils.ConcertoService) (*DomainService, er
 func (dm *DomainService) GetDomainList() (domains []types.Domain, err error) {
 	log.Debug("GetDomainList")
 
-	data, status, err := dm.concertoService.Get("/v1/dns/domains")
+	data, status, err := dm.concertoService.Get("/v2/dns/domains")
```

Comment:
Actually, these resources no longer exist. I believe we already removed the related commands, but it would be a good time to remove the dns-related services, since it makes no sense to update the version on their routes...